Project_1/Styles.css

```
@@ -1,76 +1,150 @@
-img {
-    src: "Octopus.jpg";
-    max-width: 200%;
-    height: 700px;
-    width: 110%;
-    float: left;
+/* ==========
+GENERAL
+=============*/
+
+*:before,
+*:after {
+    Box-sizing: Border-box;
```

Remember to keep your CSS lowercase:
```suggestion
    box-sizing: border-box;
```